### ISSUE TITLE
Automatic Time/Timezone in settings has no effect

### ISSUE THREAD

**Issue by LiamPM5** [pop-os/pop#1736]
Distribution : Pop! OS 20.10
 
Related Application and/or Package Version: Settings

Issue/Bug Description:
Settings app does not appear to handle automatic date and time/time zone at all, when settings are turned off then back on again in a different time zone, nothing happens
Steps to reproduce (if you know): Unknown

Expected behavior:
Turning "automatic date and time//time zone" on would be effective

Other Notes:
Traveled from UTC -8 to UTC -4 and automatic time and timezone adjustment settings did not take effect at any point, it had to be set manually after this was noticed

**Comment by jacobgkau** [comment]
Are you connected to the internet on the machine in question? Are you using a VPN that might be hiding your IP address from the geolocation service?

Does running `timedatectl set-ntp false` followed by `timedatectl set-ntp true` successfully correct your system's time?

**Comment by LiamPM5** [comment]
Yes, I am connected to the internet when I attempt to use the auto timezone. I'm not using a VPN or a Proxy of any sort. As for the commands, those also seem to have no effect on the time

**Comment by LiamPM5** [comment]
Source of issue was discovered. The IP address of where I was connecting from was erroneously reporting that it was in Seattle when it should say Indianapolis, issue does not have to do with PopOS

**Comment by bflanagin** [comment]
Might still be an issue, I happened to have family running Pop!_OS go from Colorado to Texas last weekend without the time zone switching automatically even though the option was set. I have also been able to replicate the issue on my own machine (possibly.)

**Comment by bflanagin** [comment]
The option appears to be broken upstream, will update when there is more info.

**Comment by ianhundere** [comment]
same issue here.
